Tests/Command/DropDatabaseDoctrineTest.php

```diff
@@ -42,6 +43,72 @@ public function testExecute() : void
         );
     }
 
+    public function testExecuteWithAlias() : void
+    {
+        $connectionName = 'default';
+        $dbName         = 'test';
+        $params         = [
+            'url' => 'sqlite:///' . sys_get_temp_dir() . '/test.db',
+            'path' => sys_get_temp_dir() . '/' . $dbName,
+            'driver' => 'pdo_sqlite',
+        ];
+
+        $container = $this->getMockContainer($connectionName, $params);
+
+        $application = new Application();
+        $application->add(new DropDatabaseDoctrineCommand($container->get('doctrine')));
+
+        $command = $application->find('doctrine:database:drop');
+
+        $commandTester = new CommandTester($command);
+        $commandTester->execute(
+            array_merge(['command' => $command->getName(), '-f' => true])
+        );
+
+        $this->assertContains(
+            sprintf(
+                'Dropped database %s for connection named %s',
+                sys_get_temp_dir() . '/' . $dbName,
+                $connectionName
+            ),
+            $commandTester->getDisplay()
+        );
+    }
+
+    public function testItThrowsWhenUsingIfExistsWithAnIncompatibleDriver() : void
```

Comment:
After implementing my first suggestion, I suggest to just do here
```php
public function testItThrowsWhenUsingIfExistsWithAnIncompatibleDriver() : void
{
    static::expectException(DBALException::class);
    $this->testExecute(['-f' => true, '-e' => true]);
}

Reply:
Improved 🙂